src/javascript/Api/ContentEditor.api.jsx

```diff
@@ -53,10 +54,10 @@ const ContentEditorApiCmp = ({classes, client}) => {
      * @param excludedNodeTypes (optional) The node types excluded for creation, by default: ['jmix:studioOnly', 'jmix:hiddenType']
      * @param includeSubTypes (optional) if true, subtypes of nodeTypes provided will be resolved.
      */
-    window.CE_API.create = (path, site, lang, uilang, nodeTypes, excludedNodeTypes, includeSubTypes) => {
+    window.CE_API.create = (uuid, path, site, lang, uilang, nodeTypes, excludedNodeTypes, includeSubTypes) => {
```

Comment:
There is possibility to do that ? 
```suggestion
    window.CE_API.create = ({uuid, path, site, lang, uilang, nodeTypes, excludedNodeTypes, includeSubTypes}) => {
```

Reply:
It would have been nice, but as an exposed API, it is better to know the parameters to pass